docs/source/TaskEngineWorkflows.md

```diff
@@ -348,6 +354,39 @@ This workflow allows you to create an MP4 from a VOD asset
 }
 ```
 
+## build_thumbnails
+
+This workflow allows the generation of a thumbnail sprite and vtt file, which can then be used for video timeline preview thumbnails.
+
+## build_thumbnails: Parameters
+
+| Parameter Name    | Required |  Description | Default |
+| ----------------- | -------- | ------------ | ------- |
+| workflow          |Yes| Specify 'build_thumbnails'.||
+| source_file       |Yes| file name of the asset to generate the sprite/vtt from.||
+| target_file_name  |Yes| partial file names for the generated assets, format: {target_file_name}_{sprite/vtt}.jpg .||
+| output_folder     |Yes| output folder for the created assets to be saved too
+| seconds_between   |NO | Time between thumbnail captures from source in seconds. || 60
```

Comment:
'from source' as apposed to? 😄. 

'Time between thumbnail captures in seconds'